content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/fedex.rst

```diff
@@ -0,0 +1,173 @@
+=================
+FedEx integration
+=================
+
+Integrating a FedEx account with Odoo's **Inventory** app makes it possible to :doc:`calculate
+shipping rates <delivery_method>`, and :doc:`generate shipping labels <labels>` within Odoo. This is
+accomplished by enabling the FedEx *shipping connector*, and then configuring at least one *shipping
+method*.
+
+.. note::
+   This documentation contains configuration details specific to FedEx integration. See the
+   documentation on :doc:`third-party shippers <third_party_shipper>` for general shipper
+   integration instructions.
+
+Enable shipping connector
+=========================
+
+To enable the shipping connector for FedEx, navigate to :menuselection:`Inventory app -->
+Configuration --> Settings`. Scroll down to the :guilabel:`Shipping Connectors` heading, and tick
+the checkbox next to :guilabel:`FedEx Connector`.
+
+Finally, click :guilabel:`Save` to save the changes. After doing so, a :icon:`oi-arrow-right`
+:guilabel:`FedEx Shipping Methods` button appears below :guilabel:`FedEx Connector`.
```

Comment:
```suggestion
:guilabel:`FedEx Shipping Methods` link appears below :guilabel:`FedEx Connector`.
```
I see this more as a link and less of a "button," per se. But, perhaps, that's just my opinion. 😆